backend/app/api/journal.py

```
@@ -105,6 +113,179 @@ async def parse_stt(
+        for entry in result.get("entries", []):
+            try:
+                enriched_entries.append(await enrich_with_pesticide_match(db, entry))
+            except Exception:
+                enriched_entries.append(entry)
+        result["entries"] = enriched_entries
+    except Exception:
+        pass
+
+    # 사진 디스크 저장 + DB 행 생성 (entry_id=null 임시 사진)
+    saved_ids: list[int] = []
+    for img_bytes, f in zip(images, files):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)